pkgs/development/tools/misc/strace/default.nix

```diff
@@ -1,5 +1,7 @@
 { lib, stdenv, fetchurl, perl, libunwind, buildPackages }:
 
+assert stdenv.isLinux;
```

Comment:
Can we add a comment explaining the situation?

Reply:
Done.